ci/gpu/build.sh

```diff
@@ -51,11 +51,11 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       cmake==3.14.3 \
       umap-learn \
       nccl>=2.4 \
-      dask \
-      distributed \
+      dask=2.3.0 \
+      distributed=2.3.0 \
```

Comment:
Does cuml directly depend on `distributed`, or should this be pulled in via `dask`? If the latter, then we should remove `distributed` since (I think) best practice dictates transitive dependencies shouldn't be mentioned.

Reply:
This is a great point. I'm not completely sure why we have it pulling in both and I do see that the [Dask feedstock](https://github.com/conda-forge/dask-feedstock/blob/master/recipe/meta.yaml) has a direct dependency on distributed. I'll remove distributed from the env yaml/ci builds.